### DISCUSSION TITLE
Feature request: Skip Track Function

### DISCUSSION THREAD

**Discussion by bwims** [alexkn/node-red-contrib-dlna#30] [Ideas]
Hello,

I'm guessing that this will not be accepted since it's been 2 years now, but this seems such a useful thing that I thought I'd ask:

I have a dlna hardware renderer plugged into a hifi system and once a selection of music is queued I would like to pause, resume, stop and **skip a track**.

I have a wireless handheld remote control with 4 buttons which I use with a Sonoff RF Bridge which I would like to use to avoid having to go to a computer/use a phone to skip tracks.

If you would consider adding this command I'd be truly grateful!

B.

**Comment by bwims** [comment]
Whoops!  It turns out that in this context, stop actually works like skip forward!  (Skip back would be great though!)

**Comment by alexkn** [comment]
There are no such functions in the underlying [upnp-mediarenderer-client](https://www.npmjs.com/package/upnp-mediarenderer-client) library.

**Reply by bwims** [reply]
I see!  Oh well, although it might be interesting to play with the other functions, I understand!

Thanks!

B.
